cmd/txtar-addmod/addmod.go

```diff
@@ -36,6 +36,22 @@ import (
 func usage() {
 	fmt.Fprintf(os.Stderr, "usage: txtar-addmod dir path@version...\n")
 	flag.PrintDefaults()
+
+	fmt.Fprintf(os.Stderr, `
+The txtar-addmod command adds a module as a txtar archive to the a
+testdata module directory as understood by the goproxytest package
+(see https://godoc.org/github.com/rogpeppe/go-internal/goproxytest).
+
+The dir argument names to directory to add the module to. If dir is "-",
```

Comment:
"names _the_ directory"

Reply:
Also, I'm aware of the use of `-` as stdin, but stdout?

Reply:
Yeah, it's not uncommon (the `tar` command uses that convention for example).